File: WeAre.ReApptor.React.Toolkit/src/models/GeoCoordinate.ts
export default class GeoCoordinate {
    constructor(latitude: number | null | undefined = null, longitude: number | null | undefined = null) {
        this.lat = latitude || 0.0;
        this.lon = longitude || 0.0;
    }

    public lat: number = 0.0;

    public lon: number = 0.0;

    public isGeoCoordinate: boolean = true;
}

File: WeAre.Apps.Common/Models/Geo/GeoCoordinate.cs
using WeAre.Apps.Common.Interfaces.Geo;

namespace WeAre.Apps.Common.Models.Geo
{
    public class GeoCoordinate : IGeoCoordinate
    {
        /// <summary>
        /// "Latitude"
        /// </summary>
        public decimal Lat { get; set; }

        /// <summary>
        /// "Longitude"
        /// </summary>
        public decimal Lon { get; set; }
    }
}